contrib/devtools/bitcoin-tidy/CMakeLists.txt

```
@@ -8,7 +8,11 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_CXX_EXTENSIONS False)
 
-# TODO: Figure out how to avoid the terminfo check
+set(CMAKE_DISABLE_FIND_PACKAGE_CURL ON)
+set(CMAKE_DISABLE_FIND_PACKAGE_ZLIB ON)
+
+set(LLVM_ENABLE_TERMINFO OFF)
```

Why not using the same approach:
```suggestion
set(CMAKE_DISABLE_FIND_PACKAGE_Terminfo ON)
```
?

---

Mostly because I assumed we'd want to use the LLVM options, happy to change either way.